keras_trainer/data_generators.py

```diff
@@ -26,6 +26,7 @@ class EnhancedBatchFromFilesMixin(BatchFromFilesMixin):
     def set_processing_attrs(self,
                              image_data_generator,
                              random_crop_size,
+                             custom_crop,
```

Comment:
The same way you're referencing `custom_crop` before `random_crop_size` in `_get_batches_of_transformed_samples()`, by not defining it here before too?
```suggestion
                             custom_crop,
                             random_crop_size,
```